OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 8

# Description
Lint fix for Reports component, this is phase 8.

## Files changed
All js/jsx file in /src/components/Reports/TasksDetail

eslint test:
<img width="1556" alt="CleanShot 2023-09-30 at 18 50 33@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/8224613e-18ce-4987-ba49-c84cfddf02e4">

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint --fix src/components/Reports/TableFilter/index.js`
- `npx eslint --fix src/components/Reports/TableFilter/TableFilter.jsx`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
